oca/tests/test_api_foreach.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import unittest
+
+from .. import api
```

Comment:
Do you think it's a good idea naming the module in a way that will conflict with Odoo's?
Yes, I know you can always `from oca import api as _api` or `from oca.api import foreach`, but I feel we'd be better avoiding the conflict. Not sure what name to use instead, though.
Or is your idea that methods get decorated as `@oca.api.foreach()` instead?